lang/rust/avro/src/error.rs

```diff
@@ -478,6 +478,42 @@ pub enum Error {
 
     #[error("Invalid Avro data! Cannot read codec type from value that is not Value::Bytes.")]
     BadCodecMetadata,
+
+    #[error("Schemas are not compatible. '{0}'")]
+    CompatibilityError(String),
```

Comment:
This should be either removed (preferred I think), or propagte from the other enum:

```rs
CompatibilityError {
  #[from]
  source: CompatibilityError
},
```

Reply:
Oh yes!! It is not used at all!